### ISSUE TITLE
[NEWS] Zamówienia - 28 sierpnia 2024 dodamy nowe typy identyfikatora podatkowego / Orders - on August 28, 2024, we will add new tax ID types

### ISSUE THREAD

**Issue by AureliuszBrussy** [allegro/allegro-api#9650]
**28 sierpnia 2024** dodamy obsługę nowych typów identyfikatora podatkowego.

**Jakie zmiany wprowadzimy?**

Zmian dotyczy: 

- [GET /order/checkout-forms](https://developer.allegro.pl/documentation#operation/getListOfOrdersUsingGET)
- [GET /order/checkout-forms/{id}](https://developer.allegro.pl/documentation#operation/getOrdersDetailsUsingGET)

Począwszy od **28 sierpnia**, rozszerzymy słownik dla pola **invoice.address.company.ids[].type** o nowe wartości:

- HU_ADOSZAM
- SK_ICO
- SK_IC_DPH

Miej na uwadzę, że ze względu na czas rozpropagowania zmiany, przez kilka tygodni po 28 sierpnia, dla kupujących z Węgier oraz Słowacji, obok identyfikatorów podatkowych o typie HU_ADOSZAM, SK_ICO oraz SK_IC_DPH **nadal będziemy przekazywać identyfikatory podatkowe** o typie OTHER.

Dodatkowo, jeżeli kupujący ze Słowacji nie podał w procesie zakupowym numeru SK_IC_DPH, to opcja deklaracji płatnika VAT nie jest wtedy dostępna, w związku z tym w polu **invoice.address.company.vatPayerStatus** zwrócimy wartość NOT_APPLICABLE.

Więcej o obsłudze zamówień w ramach publicznego API przeczytasz w [naszym poradniku](https://developer.allegro.pl/tutorials/jak-obslugiwac-zamowienia-GRaj0qyvwtR).
__________

On **August 28, 2024**, we will add support for new tax ID types.

**What changes will we make?**

The changes apply to:  

- [GET /order/checkout-forms](https://developer.allegro.pl/documentation#operation/getListOfOrdersUsingGET)
- [GET /order/checkout-forms/{id}](https://developer.allegro.pl/documentation#operation/getOrdersDetailsUsingGET)

Starting **August 28**, we will expand the dictionary for the **invoice.address.company.ids[].type** field with new values:

- HU_ADOSZAM
- SK_ICO
- SK_IC_DPH

Please note that due to the change propagation timing, for a few weeks after August 28, for buyers from Hungary and Slovakia, **we will continue to provide tax identifiers** of the OTHER type in addition to the tax identifiers of the HU_ADOSZAM, SK_ICO, and SK_IC_DPH types.

In addition, if the Slovak buyer did not provide the SK_IC_DPH number in the purchasing process, the VAT payer declaration option is unavailable, so the **invoice.address.company.vatPayerStatus** field will return NOT_APPLICABLE.

You can read more about orders processing via the public API [in our guide](https://developer.allegro.pl/tutorials/process-orders-PgPMlWDr8Cv).

**Comment by ZajdaBear** [comment]
Hi, is thre a list of all the types used on Allegro? So far I am aware of SK_ICO, SK_IC_DPH, CZ_ICO, CZ_DIC and HU_ADOSZAM

Thanks

**Comment by Lukasz-Zurek** [comment]
@ZajdaBear Yes, it is up to date for now. If any changes occur, we will inform about it in [our news](https://developer.allegro.pl/news) and on Github.

**Comment by Lukasz-Zurek** [comment]
@ZajdaBear Sorry, I forgot about VAT_EU. Full list is available in our documentation:

- [GET /order/checkout-forms](https://developer.allegro.pl/documentation#operation/getListOfOrdersUsingGET)
- [GET /order/checkout-forms/{id}](https://developer.allegro.pl/documentation#operation/getOrdersDetailsUsingGET)

**Comment by ZajdaBear** [comment]
Thanx, I was able to find it this time :)
